VRCFaceTracking/OSC/OSCMain.cs

```diff
@@ -70,8 +51,29 @@ private void Recv()
                 return;
             }
             var newMsg = new OscMessage(buffer);
-            if (newMsg.Address == "/avatar/change")
-                ConfigParser.ParseNewAvatar((string) newMsg.Value);
+            switch (newMsg.Address)
+            {
+                case "/avatar/change":
+                    ConfigParser.ParseNewAvatar((string)newMsg.Value);
+                    break;
+                case "/avatar/parameters/EyeTrackingActive":
+                    if (UnifiedLibManager.EyeStatus != ModuleState.Uninitialized)
+                    {
+                        if (!(bool)newMsg.Value)
+                            UnifiedLibManager.EyeStatus = ModuleState.Idle;
+                        else UnifiedLibManager.EyeStatus = ModuleState.Active;
+                    }
+                    break;
+                case "/avatar/parameters/LipTrackingActive":
+                case "/avatar/parameters/ExpressionTrackingActive":
+                    if (UnifiedLibManager.ExpressionStatus != ModuleState.Uninitialized)
+                    {
+                        if (!(bool)newMsg.Value)
+                            UnifiedLibManager.ExpressionStatus = ModuleState.Idle;
+                        else UnifiedLibManager.ExpressionStatus = ModuleState.Active;
+                    }
+                    break;
+            }
```

Comment:
I wonder if we could replace this in exchange for another class for handling OSC toggle types derived from a simple `BoolBaseParam`?

Reply:
Maybe we could flesh out this BoolBaseParam derivitive I added to work around the LipTrackingActive and EyeTrackingActive parameters, and allow it to accept unique func delegates that can then be called, or it could be it's own class:  [ConditionalBoolParameter](https://github.com/benaclejames/VRCFaceTracking/pull/109/files/5d7d16c561782b3f5c037707cd71b3e03d157d01#diff-ac24c6fa2cf66c9ea74377c07d568956a9bf661364132cc419604b0b9d5cb07eR43)?

It could be set in it's own container (eg. Toggles) that are only used to change states within VRCFT and not intended for output parameters, since we aren't really expecting output parameters to be controlled via recieves.